src/lib/database/store.ts

```diff
@@ -418,16 +418,16 @@ const initializer = immer<Database>((set, get) => ({
 
   simulateWorkspaceOutage(workspace_id, { routes }) {
     set((state) => {
-      state.simulatedWorkspaceOutages.set(workspace_id, {
+      state.simulatedWorkspaceOutages[workspace_id] = {
         workspace_id,
         routes,
-      })
+      }
     })
   },
 
   simulateWorkspaceOutageRecovery(workspace_id) {
     set((state) => {
-      state.simulatedWorkspaceOutages.delete(workspace_id)
+      state.simulatedWorkspaceOutages[workspace_id] = undefined
```

Comment:
Can this be deleted from the object instead of set undefined?

Reply:
There is a lint rule disallowing deletion

Reply:
This bug is blocking some people so I created an issue to discuss the lint rule and "fixing" the deletion

https://github.com/seamapi/fake-seam-connect/issues/83